docs/potato_paper.md

```diff
@@ -0,0 +1,111 @@
+# Potato Paper
+
+## Proof-of-Potato
+
+Proof-of-Potato (PoP) is a protocol for running decentralized sharded off-chain state machines.
+PoP is managed by a potato smart contract hosted on a secure blockchain such as Ethereum.
+The potato smart contract manages potato shard ownership and list of potato verifiers.
+Each potato shard runs a blockchain based potato state machine using our novel Proof-of-Tater (PoT) potato consensus algorithm.
```

Comment:
Perhaps we should not mention protocols not yet designed? Or at least say here is what it is supposed to do:...

Reply:
It's just reads weirdly for me I guess

Reply:
I guess I'm  still confused on what PoT actually is. Add a list of things handled by PoT? Like cross-shard, rewards, verfier assignment

Reply:
PoT is your original PoP because I want to use PoP to refer to the whole thing, not just the consensus protocol on a given shard. Does that make sense?

We don't have to do that. We can come up with a different name.

Reply:
Oh I see. I had the two mixed together

Reply:
Does it mean I have to rename my file :terror:  :terror:  :terror:  :terror: :terror:

Reply:
DDD: